src/Sarif.Converters/PyLintObjectModel/PyLintLogReader.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace Microsoft.CodeAnalysis.Sarif.Converters.PyLintObjectModel
+{
+    public static class PyLintLogReader
```

Comment:
The methods within this class seem to have a similar signature to those within TSLintLoader.cs.  It could be a good idea to make these methods part of some generic abstract base class or interface.

Reply:
@jeff-bacon Hmm, perhaps:
```
public abstract class LogReader<TLog> where TLog: class
{
    public static TLog ReadLog(string input)
    {
        return ReadLog<TLog>(input, Encoding.UTF8);
    }

    public static TLog ReadLog(string input, Encoding encoding)
    {
        return ReadLog<TLog>(new MemoryStream(encoding.GetBytes(input)));
    }

    public static TLog ReadLog(Stream input)
    {
        var reader = new LogReader<TLog>();
        return reader.ReadLogCore(input);
    }

    protected abstract TLog ReadLogCore(Stream input);
}

public PyLintLogReader : LogReader<PyLintLog>
{
    // Implement ReadLogCore(Stream)
}
```

It's